docs/first_pr.md

```diff
@@ -0,0 +1,61 @@
+# Getting started with EvalAI Issues:
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [this](https://github.com/Cloud-CV/EvalAI) link.
+
+**Step 2 : Picking up a issue:**
+
+1. Pick up a suitable issue that you find will be facile for you to fix. Moreover, you can also 
+take the issues based on their labels. Almost every issue is marked with a label according to its difficulty.
+
+2. After selecting any issue, mention any mentor’s name there and ask him to assign it to you.
+
+3. Once it gets assigned , create a branch from your fork’s updated master branch using the command :-
```

Comment:
Can we make it `using the following command.` only.

Also you can actually remove `:-` everywhere from document and simply address by _following_ or any suitable word.